# Study 849 — Dry January / Veganuary — the teardown

The January/February abnormal returns, the Newey-West January-dummy *t*, the twelve-month calendar placebo, the two-era robustness cut, the costed timer, and the 20-seed synthetic control.

In [1]:
R = {'start': '1999-01-04', 'end': '2026-06-30', 'as_of': '2026-06-30', 'fingerprint': 'e5210651db30', 'n_rows': 6914, 'n_tickers': 8, 'alc_jan_n': 27, 'alc_jan_mean': -1.2, 'alc_jan_t1s': -1.28, 'alc_jan_tnw': -1.46, 'alc_jan_hit': '10/27', 'plant_jan_n': 7, 'plant_jan_mean': 12.71, 'plant_jan_t1s': 1.23, 'plant_jan_tnw': 1.82, 'plant_jan_hit': '5/7', 'stap_jan_n': 27, 'stap_jan_mean': -0.62, 'stap_jan_t1s': -0.8, 'spread_jan_n': 7, 'spread_jan_mean': 14.46, 'spread_jan_t1s': 1.33, 'spread_jan_tnw': 1.76, 'spread_jan_hit': '5/7', 'alc_feb_mean': 0.12, 'alc_feb_t1s': 0.13, 'alc_placebo_rank': 1, 'alc_placebo_p': 0.083, 'plant_placebo_rank': 1, 'plant_placebo_p': 0.083, 'era_early_n': 13, 'era_early_mean': -1.58, 'era_early_t': -1.37, 'era_late_n': 14, 'era_late_mean': -0.85, 'era_late_t': -0.57, 'timer5_gross': 14.46, 'timer5_cost': 0.24, 'timer5_net': 14.22, 'timer5_t': 1.31, 'timer10_net': 14.02, 'timer10_t': 1.29, 'null_t': -0.2, 'null_fire': 0, 'edge2_t': 1.39, 'edge2_fire': 20, 'edge5_t': 3.77, 'edge5_fire': 90}

## The headline — January abnormal return (group − SPY), one obs/year

In [2]:
print(f"alcohol         (n={R['alc_jan_n']:>2}) : {R['alc_jan_mean']:+.2f}%  "
      f"t_1s = {R['alc_jan_t1s']:+.2f}  NW t = {R['alc_jan_tnw']:+.2f}  hit {R['alc_jan_hit']}")
print(f"plant  (BYND)   (n={R['plant_jan_n']:>2}) : {R['plant_jan_mean']:+.2f}%  "
      f"t_1s = {R['plant_jan_t1s']:+.2f}  NW t = {R['plant_jan_tnw']:+.2f}  hit {R['plant_jan_hit']}")
print(f"staples (XLP)   (n={R['stap_jan_n']:>2}) : {R['stap_jan_mean']:+.2f}%  "
      f"t_1s = {R['stap_jan_t1s']:+.2f}   <- flat control")
print(f"plant - alcohol (n={R['spread_jan_n']:>2}) : {R['spread_jan_mean']:+.2f}%  "
      f"t_1s = {R['spread_jan_t1s']:+.2f}  NW t = {R['spread_jan_tnw']:+.2f}  hit {R['spread_jan_hit']}")

alcohol         (n=27) : -1.20%  t_1s = -1.28  NW t = -1.46  hit 10/27
plant  (BYND)   (n= 7) : +12.71%  t_1s = +1.23  NW t = +1.82  hit 5/7
staples (XLP)   (n=27) : -0.62%  t_1s = -0.80   <- flat control
plant - alcohol (n= 7) : +14.46%  t_1s = +1.33  NW t = +1.76  hit 5/7


## Calendar placebo — is January special, or one of twelve months?

Rank January's mean abnormal return among all twelve calendar months (directional).

In [3]:
print(f"alcohol (left tail) : Jan is rank {R['alc_placebo_rank']}/12  -> p = {R['alc_placebo_p']:.3f}")
print(f"plant   (right tail): Jan is rank {R['plant_placebo_rank']}/12  -> p = {R['plant_placebo_p']:.3f}")
print('Both land January at rank 1/12 in the predicted direction — but a pre-specified')
print('month placebo can do no better than p = 1/12 = 0.083, which misses 0.05.')

alcohol (left tail) : Jan is rank 1/12  -> p = 0.083
plant   (right tail): Jan is rank 1/12  -> p = 0.083
Both land January at rank 1/12 in the predicted direction — but a pre-specified
month placebo can do no better than p = 1/12 = 0.083, which misses 0.05.


## Robustness — alcohol January abnormal return, two eras (split 2013)

In [4]:
print(f"1999-2012 (n={R['era_early_n']}): {R['era_early_mean']:+.2f}%  t = {R['era_early_t']:+.2f}")
print(f"2013-2026 (n={R['era_late_n']}): {R['era_late_mean']:+.2f}%  t = {R['era_late_t']:+.2f}")
print('Right sign in both halves, but fades to t = -0.57 in the modern era.')

1999-2012 (n=13): -1.58%  t = -1.37
2013-2026 (n=14): -0.85%  t = -0.57
Right sign in both halves, but fades to t = -0.57 in the modern era.


## The timer — long-plant / short-alcohol every January, costed

Enter December close, exit January close; 2 legs × round-trip + short borrow.

In [5]:
for tag,g,c,n,t in [('5 bps',R['timer5_gross'],R['timer5_cost'],R['timer5_net'],R['timer5_t']),
                    ('10 bps',R['timer5_gross'],R['timer5_cost']*2,R['timer10_net'],R['timer10_t'])]:
    print(f"{tag:>6} one-way: gross {g:+.2f}% -> net {n:+.2f}%/yr (t = {t:+.2f}, only 7 obs)")

 5 bps one-way: gross +14.46% -> net +14.22%/yr (t = +1.31, only 7 obs)
10 bps one-way: gross +14.46% -> net +14.02%/yr (t = +1.29, only 7 obs)


## Synthetic positive control — the machinery is unbiased

Live: the detector must NOT fire on the null and must recover a planted seasonal.

In [6]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
import numpy as np
from dry_january import data, strategy as st
null_t = np.array([st.synthetic_detect(data.synthetic_panel(edge=0.0, seed=849+s), month=1)['t_nw'] for s in range(8)])
print(f"null (edge=0), 8 seeds : NW t mean {null_t.mean():+.2f} (sd {null_t.std(ddof=1):.2f}), |t|>=2 in {(abs(null_t)>=2).sum()}/8")
planted = st.synthetic_detect(data.synthetic_panel(edge=0.05, seed=849), month=1)
print(f"planted (edge=0.05)    : NW t = {planted['t_nw']:+.2f}, beta = {planted['beta_pct']:+.2f}%")

null (edge=0), 8 seeds : NW t mean -0.47 (sd 0.81), |t|>=2 in 0/8


planted (edge=0.05)    : NW t = +4.09, beta = +6.27%


## Verdict

- **Signal — Weak.** The Dry-January / Veganuary footprint is *directionally perfect* — alcohol is the year's most-negative calendar month (-1.20%, rank 1/12) and `BYND` the most-positive (rank 1/12), sign-consistent across both eras — but the only well-sampled leg reaches just *t* = -1.28 (27 Januaries), the plant/spread side rides 7 noisy `BYND` years (*t* = +1.33), the February hangover is absent, and **no cut clears |*t*| ≥ 2**. The 20-seed control fires 90% on a genuine +5% seasonal and 0/20 on the null, so the whiff is real-but-sub-threshold.
- **Tradability — Mirage.** The long-plant / short-alcohol January timer nets **+14.22%/yr** at 5 bps — but on 7 observations at *t* = +1.31, driven by one small-cap's early tape, capacity-trivial and inside a multi-cut search. Nothing bankable.